onnxruntime/test/automl_ops/datetimetransformer_test.cc

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "gtest/gtest.h"
+#include "test/providers/provider_test_utils.h"
+
+#include "core/automl/featurizers/src/FeaturizerPrep/Featurizers/DateTimeFeaturizer.h"
+
+namespace dft = Microsoft::Featurizer::DateTimeFeaturizer;
+
+using SysClock = std::chrono::system_clock;
+
+namespace onnxruntime {
+namespace test {
+
+TEST(DateTimeFeaturizer_DateTime, Past_1976_Nov_17__12_27_04) {
+
+  const time_t date = 217081624;
+  OpTester test("DateTimeTransformer", 1, onnxruntime::kMSAutoMLDomain);
+
+  // We are adding a scalar Tensor in this instance
+  test.AddInput<int64_t>("X", {1}, {date});
```

Comment:
Is {1} the shape? If so, I think it is better to use the empty shape {} for a scalar (which is the recommended encoding).